opal/runtime/opal_params.c

```diff
@@ -61,6 +61,10 @@ char *opal_timing_output = NULL;
 bool opal_timing_overhead = true;
 #endif
 
+#if OPAL_ENABLE_DEBUG
+bool opal_debug_threads;
```

Comment:
What is this?  It's declared here, but it doesn't not appear to be used...?

Reply:
I believe it’s used. Let me get to my hotel and I’ll check.

Reply:
@jsquyres It's used in the following places:
```
opal/mca/threads/threads.h:61:        if (opal_debug_threads) {                        \
opal/mca/threads/threads.h:68:        if (opal_debug_threads) {                        \
opal/mca/threads/threads.h:89:        if (opal_debug_threads) {                       \
opal/runtime/opal_params.c:65:bool opal_debug_threads;
opal/runtime/opal_params.c:190:    opal_debug_threads = false;
opal/runtime/opal_params.c:196:                          &opal_debug_threads);
orte/util/threads.h:69:        if (opal_debug_threads) {                               \
orte/util/threads.h:76:        if (opal_debug_threads) {                               \
orte/util/threads.h:100:        if (opal_debug_threads) {                               \
orte/util/threads.h:107:        if (opal_debug_threads) {                               \
orte/util/threads.h:130:        if (opal_debug_threads) {                       \
```

Reply:
Ah, ok.  It was a little odd to see it be a "+" in a patch but not have it show up elsewhere in the patch.